### PR TITLE
Restart your shell session

### PR DESCRIPTION
**Pull request by ghost**
according to: https://github.com/Linuxbrew/brew/issues/711

### CODE REVIEW

File: install [hunk 1]
```diff
@@ -383,13 +383,13 @@ Dir.chdir HOMEBREW_REPOSITORY do
     curl_flags = "fsSL"
     curl_flags += "k" if mac? && macos_version < "10.6"
     core_tap = "#{HOMEBREW_PREFIX}/Homebrew/Library/Taps/homebrew/homebrew-core"
-    system "/bin/bash -o pipefail -c 'curl -#{curl_flags} #{BREW_REPO}/tarball/master | /bin/tar xz -m --strip 1'"
+    system "/bin/bash -o pipefail -c '/usr/bin/curl -#{curl_flags} #{BREW_REPO}/tarball/master | /bin/tar xz -m --strip 1'"
```
Comment (sjackman) [inline install]: Please revert this unnecessary change.

File: install [hunk 2]
```diff
@@ -383,13 +383,13 @@ Dir.chdir HOMEBREW_REPOSITORY do
...
 
     system "ln", "-sf", "#{HOMEBREW_REPOSITORY}/bin/brew", "#{HOMEBREW_PREFIX}/bin/brew" unless HOMEBREW_REPOSITORY == HOMEBREW_PREFIX
 
     system "/bin/mkdir", "-p", core_tap
     Dir.chdir core_tap do
-      system "/bin/bash -o pipefail -c 'curl -#{curl_flags} #{CORE_TAP_REPO}/tarball/master | /bin/tar xz -m --strip 1'"
+      system "/bin/bash -o pipefail -c '/usr/bin/curl -#{curl_flags} #{CORE_TAP_REPO}/tarball/master | /bin/tar xz -m --strip 1'"
```
Comment (sjackman) [inline install]: Ditto.

File: install [hunk 3]
```diff
@@ -432,6 +432,7 @@ if !mac?
     PATH="#{HOMEBREW_PREFIX}/bin:$PATH"
 - We recommend that you install GCC by running:
     brew install gcc
+- After modifying your shell profile, you may need to restart session (logout & then log back in) if the brew command isn't found.
```
Comment (sjackman) [inline install]: Change `restart session` to `restart your session` and `&` to `and`.

**Review by sjackman — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
now I think it fixed! @sjackman

File: install [hunk 4]
```diff
@@ -384,7 +384,7 @@ Dir.chdir HOMEBREW_REPOSITORY do
     curl_flags += "k" if mac? && macos_version < "10.6"
     core_tap = "#{HOMEBREW_PREFIX}/Homebrew/Library/Taps/homebrew/homebrew-core"
     system "/bin/bash -o pipefail -c 'curl -#{curl_flags} #{BREW_REPO}/tarball/master | /bin/tar xz -m --strip 1'"
-
+
```
Comment (sjackman) [inline install]: Almost there, @Bonsitar ! You've added some additional white space here at line 387. Please remove them.

File: install [hunk 5]
```diff
@@ -432,6 +432,7 @@ if !mac?
     PATH="#{HOMEBREW_PREFIX}/bin:$PATH"
 - We recommend that you install GCC by running:
     brew install gcc
+- After modifying your shell profile, you may need to restart your session (logout and then log back in) if the brew command isn't found.
```
Comment (maxim-belkin) [inline install]: I'd suggest breaking this line somewhere as it is too long.

Comment (sjackman) [reply install]: Let's add a line break after `session`.
```
- After modifying your shell profile, you may need to restart your session
  (logout and then log back in) if the brew command isn't found.
```

**Comment by ghost** [comment]
@maxim-belkin  @sjackman 
Now is this OK?

File: install [hunk 6]
```diff
@@ -432,6 +431,8 @@ if !mac?
     PATH="#{HOMEBREW_PREFIX}/bin:$PATH"
 - We recommend that you install GCC by running:
     brew install gcc
+- After modifying your shell profile, you may need to restart your session
+(logout and then log back in) if the brew command isn't found.
```
Comment (maxim-belkin) [inline install]: I'd move `(logout` under `After`
```
- After modifying...
  (logout
```

File: install [hunk 7]
```diff
@@ -384,7 +384,6 @@ Dir.chdir HOMEBREW_REPOSITORY do
     curl_flags += "k" if mac? && macos_version < "10.6"
     core_tap = "#{HOMEBREW_PREFIX}/Homebrew/Library/Taps/homebrew/homebrew-core"
     system "/bin/bash -o pipefail -c 'curl -#{curl_flags} #{BREW_REPO}/tarball/master | /bin/tar xz -m --strip 1'"
-
```
Comment (sjackman) [inline install]: Please don't delete the blank line here. The empty blank line is desirable. Previously you also had spaces on this line.

Comment (ghost) [reply install]: @sjackman Oops! 
Why? As far as I know there is no deference while the command will skip that line! 
But I think it fixed now!

File: install [hunk 8]
```diff
@@ -432,6 +432,8 @@ if !mac?
     PATH="#{HOMEBREW_PREFIX}/bin:$PATH"
 - We recommend that you install GCC by running:
     brew install gcc
+- After modifying your shell profile, you may need to restart your session
+  (logout and then log back in) if the brew command isn't found.
```
Comment (maxim-belkin) [inline install:436]: brew -> `brew`

**Comment by sjackman** [comment]
Merged. Thank you for your first contribution to Linuxbrew, Mohammad! 🏅

**Comment by ghost** [comment]
Yaaaay!!! Thanks for your helps to merge this pull request in a right way!

**Comment by sjackman** [comment]
Our pleasure!
